diamm/serializers/serializers.py

```diff
@@ -23,3 +23,16 @@ def __init__(self, *args, **kwargs):
         super(ContextDictSerializer, self).__init__(*args, **kwargs)
         if 'context' in kwargs:
             self.context = kwargs['context']
+
+class HyperlinkedContextSerializer(ContextSerializer):
+    """
+        Provides a method for getting the models related url
+    """
+    def __init__(self, *args, **kwargs):
+        super(HyperlinkedContextSerializer, self).__init__(*args, **kwargs)
+        if 'context' in kwargs:
```

Comment:
So if this is instantiated without a context it will still instantiate fine, but will fail when trying to generate the URL? That seems tricky to debug.